app/views/assessor_interface/application_forms/index.html.erb

```diff
@@ -63,6 +63,12 @@
         <% end %>
       <% end %>
     </section>
+
+    <section id="app-applications-show-all-applicants">
+      <%= f.govuk_check_boxes_fieldset :display, legend: { size: "s" }, small: true do %>
+        <%= f.govuk_check_box :display, :show_all, label: { text: "All applications" }, checked: @view_object.filter_form.display&.include?("show_all") %>
```

Comment:
```suggestion
      <%= f.govuk_check_boxes_fieldset :show_all, legend: { size: "s" }, multiple: false, small: true do %>
        <%= f.govuk_check_box :show_all, :true, multiple: false, label: { text: "All applications" }, checked: @view_object.filter_form.show_all == "true" %>
```

Thinking about the naming of these fields, I think it makes sense to stick with the `show_all` name as `display` is a little too generic in my opinion, unclear exactly what that might be.